python/mxnet/gluon/contrib/estimator/event_handler.py

```diff
@@ -191,18 +287,23 @@ class CheckpointHandler(EventHandler):
 
     def __init__(self,
                  filepath,
-                 monitor='val_accuracy',
+                 monitor=None,
                  verbose=0,
                  save_best_only=False,
                  mode='auto',
```

Comment:
can you expand what different modes mean in the doc.

Reply:
will update, thanks!